index.js

```diff
@@ -117,6 +121,39 @@ f1.prototype = {
    */
   targets: function( targets ) {
 
+    var instance = this;
+
+    if (targets) {
+      _.forEach(targets, function (el, elementId) {
+        var elementData = {
+          id: elementId,
+          get states() {
```

Comment:
I think getters and setters are part of es5.1, just checked that it works in ie9